app/services/summary_service.py

```
@@ -0,0 +1,174 @@
+
+                if result.modified_count > 0:
+                    success_count += 1
+                    logger.debug(f"[SummaryService] Updated summary for {paper_id}")
+                else:
+                    failed_count += 1
+                    errors.append(f"Failed to update {paper_id}")
+
+            except PyMongoError as e:
+                failed_count += 1
+                error_msg = f"DB error for {paper_id}: {e}"
+                logger.error(f"[SummaryService] {error_msg}")
```

Ruff (0.14.7)

Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)